From 64a7c3612852134795543a6ea809a178abfa98d5 Mon Sep 17 00:00:00 2001
From: Aelin <aelin@postmarketos.org>
Date: Sat, 12 Sep 2026 13:00:04 +0200
Subject: [PATCH 06/10] avahi: allow NNP domain transition

Allow init_t to transition to avahi_t with NoNewPrivileges enabled.

See https://github.com/avahi/avahi/issues/867
---
 policy/modules/contrib/avahi.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/contrib/avahi.te b/policy/modules/contrib/avahi.te
index cd83e4392..c5e73c4f4 100644
--- a/policy/modules/contrib/avahi.te
+++ b/policy/modules/contrib/avahi.te
@@ -8,6 +8,7 @@ policy_module(avahi, 1.14.1)
 type avahi_t;
 type avahi_exec_t;
 init_daemon_domain(avahi_t, avahi_exec_t)
+init_nnp_daemon_domain(avahi_t)
 
 type avahi_initrc_exec_t;
 init_script_file(avahi_initrc_exec_t)
-- 
2.55.0

